index.bs

```diff
@@ -262,7 +267,16 @@ EventData = (
 
 ## Session ## {#session}
 
-WebDriver BiDi uses the same [=/session=] concept as WebDriver.
+WebDriver BiDi extends the [=/session=] concept from WebDriver.
+
+A [=/session=] has a <dfn>BiDi flag</dfn>, which is false unless otherwise
+stated.
+
+A <dfn>WebDriver BiDi Session</dfn> is a [=/session=] which has the [=BiDi flag=]
+set to true.
+
+An implementation has a list of <dfn>active BiDI sessions</dfn>, which is
```

Comment:
It looks like nothing is ever added to this list. Rather than adding that bookkeeping, however, can we just handwave "all WebDriver BiDi sessions" or something?

Typo fix if we keep it:

```suggestion
An implementation has a list of <dfn>active BiDi sessions</dfn>, which is
```